### PR TITLE
Add CheMut workflow

### PR DESCRIPTION
**Pull request by jkokosar**
@tjanez vc-chemut process requires non-standard Docker image. Please advise how to approach the related tests.

### CODE REVIEW

**Comment by mstajdohar** [comment]
Rebase on to master

**Comment by jkokosar** [comment]
@tjanez Rebased and tested. Please review.

File: resolwe_bio/utils/filter.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""Test filter functions."""
+
+
+def filter_vcf_variable(line):
```
Comment (jkokosar) [inline resolwe_bio/utils/filter.py]: @tjanez This is new since Miha's review. Please check if it is ok.

**Comment by mstajdohar** [comment]
Rebase

File: resolwe_bio/utils/filter.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    """Filter variable lines from the VCF file."""
+    if line.startswith(b"##samtoolsVersion"):
+        return True
+    elif line.startswith(b"##reference"):
+        return True
+    elif line.startswith(b"##fileDate"):
+        return True
+    elif b"/data_all/" in line:
+        return True
```
Comment (mstajdohar) [inline resolwe_bio/utils/filter.py]: Are we missing
```
else:
    return False
```

**Pull request merged**